plugin/hosts/hostsfile.go

```diff
@@ -252,3 +255,19 @@ func (h *Hostsfile) LookupStaticAddr(addr string) []string {
 	copy(hostsCp[len(hosts1):], hosts2)
 	return hostsCp
 }
+
+var (
+	hostsEntries = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: plugin.Namespace,
+		Subsystem: "hosts",
+		Name: "entries",
+		Help: "The number of entries in hosts file and corefile",
+	}, []string{})
+
+	hostsReloadTime = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: plugin.Namespace,
+		Subsystem: "hosts",
+		Name:      "reload_timestamp_seconds",
```

Comment:
this clashes with 'reload'; or will this also be updated?
Should probably be namaed `reload_time_seconds`. The timestamp is unix epoch seconds?

Reply:
This metric will not be upadated when reloading.  IMO timestamp is OK because prometheus also uses `_timestamp_seconds` to name the time of reload.